applications/asset_tracker/src/gps_controller/gps_controller.c

```diff
@@ -214,14 +214,14 @@ int gps_control_init(gps_trigger_handler_t handler)
 
 	gps_dev = device_get_binding(CONFIG_GPS_DEV_NAME);
 	if (gps_dev == NULL) {
-		printk("Could not get %s device\n", CONFIG_GPS_DEV_NAME);
+		LOG_ERR("Could not get %s device", CONFIG_GPS_DEV_NAME);
```

Comment:
`CONFIG_GPS_DEV_NAME` needs to be wrapped in `log_strdup()` if we want to support running without `CONFIG_LOG_IMMEDIATE` enabled